modules/ROOT/pages/implementing-apikit-4-for-soap.adoc

```diff
@@ -0,0 +1,12 @@
+= Implementing a SOAP Interface
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Create an API by using Studio and APIkit for SOAP to consume a remote WSDL API. Download this link:{attachmentsdir}/tshirt2.wsdl[WSDL tshirt2.wsdl] example to create a SOAP interface that accesses data resources and integrates the interface with backend processes that function as an API.
+
+Stat by xref:apikit-4-soap-project-task.adoc[].
```

Comment:
This doesn't make sense to me.

Reply:
If you use a link in context, add custom text to fix the capitalization. LMK if this is unclear.

Reply:
Hi, @hannanelson!
I've added a custom text to the link. Thanks!
Please, can you elaborate on the topic that does not make sense to you? Thanks!

Reply:
@Crispy-Salesforce - "Stat by xxx". Should Stat be Start?

Reply:
Great catch! Thanks!